LICENSE.txt

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-    Copyright [yyyy] [name of copyright owner]
+    Copyright 2015 Apple Inc.
```

Comment:
As the comment above this mentions, these are *instructions* on how to apply the copyright to each individual source file and this should be left as-is. Copyright is added to each individual source file.

> To apply the Apache License to your work, attach the following boilerplate notice, with the fields enclosed by brackets "[]" replaced with your own identifying information. (Don't include the brackets!)  The text should be enclosed in the appropriate comment syntax for the file format. We also recommend that a file or class name and description of purpose be included on the same "printed page" as the copyright notice for easier identification within third-party archives.